android/src/main/java/com/horcrux/svg/RNSVGDefsShadowNode.java

```diff
@@ -25,5 +25,12 @@ public boolean run(RNSVGVirtualNode node) {
                 return true;
             }
         });
+        traverseChildren(new NodeRunnable() {
+            public boolean run(RNSVGVirtualNode node) {
+                node.markUpdateSeen();
+                node.traverseChildren(this);
```

Comment:
LGTM.
But can you assign the NodeRunnable to a variable?
The `this` is a little confused.